stages/test/test_selinux.py

```diff
@@ -73,33 +109,72 @@ def test_selinux_file_contexts_exclude(mocked_setfiles, tmp_path, stage_module):
         "file_contexts": "etc/selinux/thing",
         "exclude_paths": ["/sysroot"],
     }
-    stage_module.main(tmp_path, options)
+    args = {
+        "tree": f"{tmp_path}",
+        "options": options
+    }
+    stage_module.main(args)
 
     assert len(mocked_setfiles.call_args_list) == 1
     args, kwargs = mocked_setfiles.call_args_list[0]
-    assert args == (f"{tmp_path}/etc/selinux/thing", os.fspath(tmp_path), "")
+    assert args == (f"{tmp_path}/etc/selinux/thing", os.fspath(tmp_path), "/")
     assert kwargs == {"exclude_paths": [f"{tmp_path}/sysroot"]}
 
 
 @patch("osbuild.util.selinux.setfilecon")
 @patch("osbuild.util.selinux.setfiles")
 def test_selinux_labels(mocked_setfiles, mocked_setfilecon, tmp_path, stage_module):
     testutil.make_fake_input_tree(tmp_path, {
-        "/usr/bin/bootc": "I'm only an imposter",
+        "/usr/bin/echo": "I'm only an imposter",
+        "/sbin/sulogin": "I'm only an imposter",
     })
 
     options = {
         "file_contexts": "etc/selinux/thing",
         "labels": {
-            "/tree/usr/bin/bootc": "system_u:object_r:install_exec_t:s0",
+            "tree:///usr/bin/echo": "system_u:object_r:bin_t:s0",
+            "/sbin/sulogin": "system_u:object_r:sulogin_exec_t:s0",
         }
     }
-    stage_module.main(tmp_path, options)
+    args = {
+        "tree": f"{tmp_path}/tree",
+        "options": options
+    }
+    stage_module.main(args)
 
     assert len(mocked_setfiles.call_args_list) == 1
+    assert len(mocked_setfilecon.call_args_list) == 2
+    assert mocked_setfilecon.call_args_list == [
+        call(f"{tmp_path}/tree/usr/bin/echo", "system_u:object_r:bin_t:s0"),
+        call(f"{tmp_path}/tree/sbin/sulogin", "system_u:object_r:sulogin_exec_t:s0"),
+    ]
+
+
+@patch("osbuild.util.selinux.setfilecon")
+def test_selinux_labels_mount(mocked_setfilecon, tmp_path, stage_module):
+    tree = tmp_path / "tree"
+    mounts = tmp_path / "mounts"
+
+    testutil.make_fake_tree(mounts, {"/sbin/su": "I'm only an imposter"})
+    args = {
+        "tree": f"{tree}",
```

Comment:
(super nitpick) afaict this `f""` here is not needed, just ` "tree": tree`, should work